### PR TITLE
Improve 'Observing Your Application with Choreo'

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

**Review by nadundesilva — COMMENTED** [review]
LGTM

File: learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ Follow the steps below to observe a Ballerina service without logging in to Chor
 
 ## Step 1 - Creating a Hello World Ballerina Service
 
-Create a `hello-world` service as given below and save it as `hello_world.bal`. 
+Create a hello world service as given below and save it as `hello_world.bal`.
```
Comment (praneesha) [inline learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md:24]:
```suggestion
Create a `hello world` service as given below and save it as `hello_world.bal`.
```

File: learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md [hunk 2]
```diff
@@ -21,9 +21,9 @@ Follow the steps below to observe a Ballerina service without logging in to Chor
...
 
->**Info:** Import the `ballerinax/choreo` module to the Ballerina code as given in the sample below in-order to add the Choreo extension to the executable.
+>**Info:** Import the `ballerinax/choreo` module as demonstrated in the sample below in order to add the Choreo extension to the executable.
```
Comment (praneesha) [inline learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md:26]:
```suggestion
>**Info:** Import the `ballerinax/choreo` module as demonstrated in the sample below to add the Choreo extension to the executable.
```

File: learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md [hunk 3]
```diff
@@ -60,7 +61,7 @@ Follow the steps below to add observability in the executable created by Balleri
 
 ## Step 3 - Observing the Ballerina Service
 
-1. Once the configuration file has been created, execute the command below to pass it to the Ballerina program with the `BAL_CONFIG_FILES` environment variable.
+1. Once the configuration file has been created, execute the command below to pass it to the Ballerina program with the `BAL_CONFIG_FILES` environment variable. This is not necessary if the `Config.toml` file is present in the current working directory.
```
Comment (praneesha) [inline learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md:64]:
```suggestion
1. Once the configuration file has been created, execute the command below to pass it to the Ballerina program with the `BAL_CONFIG_FILES` environment variable. 

>**Tip:** This is not necessary if the `Config.toml` file is present in the current working directory.
```

File: learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md [hunk 4]
```diff
@@ -80,24 +81,22 @@ Follow the steps below to add observability in the executable created by Balleri
     [ballerina/http] started HTTP/WS listener 0.0.0.0:9090
     ```
 
-2. Click on the URL provided. You will be directed to the Choreo portal where you can view the metrics and traces exposed via the throughput and latency graphs. 
-  >**Note:** If you haven't sent any requests to the Ballerina service, you may see a banner with the message "No request received during the selected time period".
+2. Click on the URL provided. You will be directed to the Choreo portal where you can view the metrics and traces exposed via the throughput and latency graphs.
```
Comment (praneesha) [inline learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md:84]:
```suggestion
2. Click on the URL provided. 
This directs you to the Choreo portal in which you can view the metrics and traces exposed via the throughput and latency graphs.
```

File: learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md [hunk 5]
```diff
@@ -80,24 +81,22 @@ Follow the steps below to add observability in the executable created by Balleri
...
+   >**Note:** If you haven't sent any requests to the Ballerina service, you may see a banner with the message "No request received during the selected time period".
```
Comment (praneesha) [inline learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md:85]:
```suggestion
   >**Note:** If you haven't sent any requests to the Ballerina service, you may see a banner with the message `No request received during the selected time period`.
```

Comment (kaneeldias) [reply learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md]: Fixed in 08c412d

File: learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md (hunk 4 above)
Comment (kaneeldias) [reply learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md]: Fixed in 08c412d

File: learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md (hunk 3 above)
Comment (kaneeldias) [reply learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md]: Fixed in 08c412d

File: learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md (hunk 2 above)
Comment (kaneeldias) [reply learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md]: Fixed in 08c412d

File: learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md (hunk 1 above)
Comment (kaneeldias) [reply learn/guides/observing-ballerina-programs/observing-your-application-with-choreo.md]: Fixed in 08c412d

**Review by praneesha — APPROVED** [review]

**Review by nadundesilva — APPROVED** [review]

**Pull request merged**